tools/oneclickapps/prepare_digital_ocean_one_click_app_release.py

```diff
@@ -122,7 +122,7 @@ def setup_one_click_app_installer(droplet: digitalocean.Droplet) -> None:
 def send_message(content: str) -> None:
     request = {
         "type": "stream",
-        "to": os.environ["ZULIP_STREAM_TO_SEND_BOT_MESSAGES"],
+        "to": "kandra ops",
```

Comment:
I think this probably still belongs as a configuration setting?

Reply:
I don't think this is a secret, though I can be overruled on this.  We can put it as a config setting, but not a secret (hardcoded in the workflow yml) but I don't think that accomplishes much.

Reply:
Yeah, I don't think it is, but it seems like we should be able to rename that stream via changing configuration, not code.

Reply:
Yeah. I think keeping it a secret is useful for changing the stream without touching code. But the frequency of that event happening is also very low so I guess both options are fine here.